src/bin/ion/commands/beta/generate/templates/java/class.templ

```diff
@@ -56,70 +58,36 @@ public class {{ target_kind_name }} {
                 null
             {% endif %};
         {% endfor %}
-        {% if abstract_data_type == "Value"%}
```

Comment:
(🗺️ PR tour) This template used to contain all abstract data types templates but now it separated into 3 templates. 
- class.templ (For struct type)
- scalar.templ
- sequence.templ